phoenix-core/src/main/java/org/apache/phoenix/hbase/index/metrics/IndexCheckerSource.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.phoenix.hbase.index.metrics;
+
+import org.apache.hadoop.hbase.metrics.BaseSource;
+
+/**
+ * Interface for metrics from GlobalIndexChecker
+ */
+public interface IndexCheckerSource extends BaseSource {
+    // Metrics2 and JMX constants
+    String METRICS_NAME = "PhoenixIndexChecker";
```

Comment:
Shouldn't the interface and metrics name be GlobalIndexCheckerSource and PhoenixGlobalIndexChecker as the metrics is not for local indexes.